src/ModuleResolver.ts

```diff
@@ -178,31 +181,23 @@ export class ModuleResolver implements ModuleResolverInterface {
     }
 
     if (moduleInstance) {
-      // If the instance is missing `format`, it's probably
-      // not an instance of Prettier
-      const isPrettierInstance = !!moduleInstance.format;
-      const isValidVersion =
-        moduleInstance.version &&
-        !!moduleInstance.getSupportInfo &&
-        !!moduleInstance.getFileInfo &&
-        !!moduleInstance.resolveConfig &&
-        semver.gte(moduleInstance.version, minPrettierVersion);
-
-      if (!isPrettierInstance && prettierPath) {
+      const version = await moduleInstance.import();
```

Comment:
We can't really access all properties of the module. I'd do 

```
await moduleInstance.get('version')
```

Reply:
We don't really need to access any properties other than version, so I think we can leave it as is.